contrib/thrift/ThriftConnection.cc

```diff
@@ -19,8 +18,8 @@ ThriftConnection::ThriftConnection(ThriftServer* server,
     state_(kExpectFrameSize),
     frameSize_(0)
 {
-  conn_->setMessageCallback(boost::bind(&ThriftConnection::onMessage,
-                                        this, _1, _2, _3));
+  conn_->setMessageCallback(std::bind(&ThriftConnection::onMessage,
+                                        this, muduo::_1, muduo::_2, muduo::_3));
```

Comment:
Replace `muduo::_1` with just `_1`, you may need to `using muduo::_1` at the beginning of this file.

Ditto for _2, _3 and below.

Reply:
I have changed it as you suggest.